app.js

```diff
@@ -40,12 +44,12 @@ app.post("/posts", async (req, res) => {
   await post.save();
 
   res.send(post);
-})
+});
 
 app.get("/friends/:username", async (req, res) => {
   const { username } = req.params;
   const user = await User.find({ username: username });
```

Comment:
instead of find use findOne so that way it only returns one document, and then remove the array indexing that you have next to user

Reply:
fixed